### ISSUE TITLE
Support for multiple schools

### ISSUE THREAD

**Issue by tejasannap** [kajchang/StudentVueApp-Docs#5]
Currently I have 2 schools listed in studentVUE. The current api documented only shows results for 1 of my schools (I take college classes at a different school). This seems to be an issue that all third party studentvue api's have. Is there a way to add support for multiple schools? I have tried requesting all terms available but I only get 2/7 of my classes in the raw XML.

**Comment by tejasannap** [comment]
For reference I am using a swift implementation I made using the http requests. I have not tried in other languages I assume it is the same. I have double checked that there is no more marking periods than for my main school.

**Comment by kaijchang** [comment]
@tejasannap That's interesting, I didn't even know that was possible. I'm not sure if I can test what the requests look like because I don't have access to two different schools. If you can, it would be interesting to use a proxy to look at what the requests look like and to see if there are multiple, etc.

**Comment by tejasannap** [comment]
Yep! I had the same idea. @kajchang Do you have a discord or something we can continue to chat this about. Id rather share the full html response in DM's rather than masking my info on here.

**Comment by kaijchang** [comment]
@tejasannap Yeah, I'd be happy to take a look. _kachang on Discord.

**Comment by tejasannap** [comment]
@kajchang friend rq sent. afaik this api was built from the mobile apps requests right? I have an m2 Mac so I'm going to try and get the data from the mobile app using an iOS Sim

**Comment by kaijchang** [comment]
@tejasannap I used Burp: https://portswigger.net/burp/documentation/desktop/mobile

**Comment by 2028163** [comment]
@kaijchang @tejasannap Did you guys ever figure out anything around the multiple schools thing, or how it works?

**Comment by tejasannap** [comment]
@2028163 its been awhile. I've graduated and have no longer worked on my app implementation. From what I can remember, the requests had 3 different IDs, there was `gradingPeriodGU`, `orgYearGU`, and `markPeriodGU`. What I found was that there was also a `schoolID` parameter. I think I dove into the requests for the app and found a way to query all schools on the account, then get the schoolID. From there, it was however you wanted to use it. I just queried all available schools and all courses. The issue is that each school will appear as a separate marking period.
